### PR TITLE
Creating CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by soxrepo**
Adding info on how to contribute to contao. Somehow this file is missing in the repo currently.

### CODE REVIEW

**Comment by leofeyer** [comment]
Thank you @soxrepo. Could you please move the file into a `.github` subfolder?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Include screenshots in your pull request whenever possible.
+
+## Git commit messages
+
+* Use the present tense ("Add feature" not "Added feature").
+* Use the imperative mood ("Move cursor to …" not "Moves cursor to …").
+* Reference issues and pull requests liberally.
+
+[1]: https://github.com/contao
+[2]: http://demo.contao.org
```
Comment (ausi) [inline CONTRIBUTING.md]: I think this should be HTTPS too.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Submitting issues
+
+* Use the search function to see if a similar issue has already been submitted.
+* Describe the issue in detail and include all the steps to follow in order to
+  reproduce the bug in the [online demo][2].
+* Include the version of Contao and PHP you are using.
+* Include screenshots if possible; they are immensely helpful.
+* If you are reporting a bug, please include any related error message you are
+  seeing and also check the `system/logs/error.log` file.
```
Comment (ausi) [inline CONTRIBUTING.md]: `system/logs/error.log` should be changed to `var/logs/prod/error.log` I think.

**Comment by ausi** [comment]
@Glumanda99 the CONTRIBUTING.md should get automatically detected by GitHub, a issue/PR template is not required for that AFAIK.

**Comment by leofeyer** [comment]
Thank you @soxrepo.

**Pull request merged**